packages/core/onClickOutside/index.ts

```diff
@@ -66,18 +62,13 @@ export function onClickOutside<T extends OnClickOutsideOptions>(
   }
 
   const cleanup = [
+    [].slice.call(window.document.body.children)
+      .map(el => useEventListener(el, 'click', () => {})),
```

Comment:
This feels too expensive and I am not sure about it.

Reply:
It's not. But we could optimize it. Currently, these event handlers will be attached on each use of `onClickOutside`. We could make it a one-time operation, initialize it only on iOS, and filter out `<script>`, `<link>` and `<style>` elements.